src/components/table/table.stories.mdx

```diff
@@ -6,7 +6,7 @@ import './table.scss';
 
 <Meta title="Components/Table" decorators={[withKnobs]} />
 
-# Table Object
```

Comment:
This change is not related to the Button or Icon refactor but was suggested as a change to make via PR feedback from @calebeby: https://github.com/cloudfour/cloudfour.com-patterns/pull/565#discussion_r397486212

Reply:
Oh good catch, thanks @gerardo-rodriguez , @calebeby !